src/requestor/routes/maintain.py

```diff
@@ -45,10 +45,13 @@ async def create_request(
     """
     Create a new access request.
 
+    Use the "revoke" query parameter to create a request to revoke access
+    instead of a request to grant access.
+
     If no "status" is specified in the request body, will use the configured
     DEFAULT_INITIAL_STATUS. Because users can only request access to a
-    resource once, (username, resource_path) must be unique unless past
-    requests' statuses are in FINAL_STATUSES.
+    resource once, each ("username", "resource_path") combination must be
```

Comment:
At a high level, do users think they are requesting/revoking access to a `resource` or a `policy`? If the latter is true, then this comment should mention something about the `policy` as well right?